src/redux/stores/balances/balancesSubscriber.ts

```diff
@@ -55,10 +48,38 @@ export const configureBalancesSubscriber = () => {
           collectionToken: config.collectionToken,
         })),
       ]).then(() => {
-        if (!balances.isInitialized) {
-          store.dispatch(setIsInitialized(true));
-        }
+        store.dispatch(setIsInitialized(true));
       });
     }
+
+    if (lastSucceededTransaction && lastSucceededTransaction.hash !== lastTransactionHash) {
+      lastTransactionHash = lastSucceededTransaction?.hash;
```

Comment:
lastSucceededTransaction should be defined at this point, no need for ?

Reply:
Nope, it can be undefined here.